huggle/loadingform.cpp

```diff
@@ -43,6 +43,11 @@ void LoadingForm::Info(QString text)
     this->ui->label_2->setText(text);
 }
 
+void LoadingForm::AbortLabel(QString text)
```

Comment:
The info function is used to distribute the info to both loading form and login, not to localize any stuff.

I guess that what you want is to call

```
this->ui->pushButton->setText(_l("login-abort"));
```

just insert that code to the constructor (on bottom of `LoadingForm::LoadingForm(QWidget *parent) : QDialog(parent), ui(new Ui::LoadingForm)
`)